packages/quantic/force-app/main/default/lwc/searchBox/searchBox.js

```diff
@@ -1,43 +1,75 @@
-import {LightningElement, api, track} from 'lwc';
-import TributePath from '@salesforce/resourceUrl/tributejs';
+import { LightningElement, api, track } from 'lwc';
 // @ts-ignore
-import {loadScript} from 'lightning/platformResourceLoader';
-import {registerComponentForInit, initializeWithHeadless} from 'c/headlessLoader';
+import { registerComponentForInit, initializeWithHeadless } from 'c/headlessLoader';
+
+const ENTER = 13;
+const ARROWUP = 38;
+const ARROWDOWN = 40;
 
 export default class SearchBox extends LightningElement {
   /** @type {import("coveo").SearchBoxState} */
   @track state = {
-    // @ts-ignore TODO: Check SearchBoxState typing and integration with LWC/Quantic
+    // @ts-ignore
     redirectTo: '',
     suggestions: [],
     value: '',
   };
+
   /** @type {any} */
-  @track suggestions = [];
+  get suggestions() {
+    return this.state.suggestions.map((s, index) => ({ key: index, value: s.rawValue }));
+  }
 
   /** @type {boolean} */
   @api sample = false;
   /** @type {string} */
   @api engineId;
+  /** @type {string} */
+  @api placeholder = 'Search';
 
   /** @type {import("coveo").SearchBox} */
   searchBox;
   /** @type {import("coveo").Unsubscribe} */
   unsubscribe;
-  /** @type {any} */
-  tribute;
-  tributeLoaded = false;
+  /** @type {number} */
+  selectionIndex = -1;
+  /** @type {HTMLInputElement} */
+  input;
+  /** @type {import("lwc").HTMLElementTheGoodPart} */
```

Comment:
HTMLElementTheGoodPart ? What?

Reply:
Yeah it's a lwc type that bring in stuff from HTMLElement and some additions from lwc to reference attributes of "native" web elements.
References to this type existed before this PR as well :)